web/cypress/e2e/app.cy.ts

```diff
@@ -25,13 +25,14 @@ describe("ARC Portal UI authenticated", () => {
     cy.loginAsAdmin();
     cy.visit("/profile/approved-researcher");
 
-    cy.get("#approved-researcher-agreement");
+    cy.get("#approved-researcher-agreement"); // wait for load
 
     cy.get("body").then((body) => {
-      if (body.find(".approved-researcher-agreement__text").length > 0) {
-        cy.get(".approved-researcher-agreement__text").should("be.visible");
+      console.log(body.find("#approved-researcher-agreement-text"));
```

Comment:
is this for debugging or an intentional log? if the latter maybe we change it to console.info and include a message?

Reply:
debugging. cheers – will rm